lib/sycamore/sycamore/transforms/text_extraction/__init__.py

```diff
@@ -11,3 +9,22 @@
     "easyocr": EasyOcr,
     "pdfminer": PdfMinerExtractor,
 }
+
+
+def TextExtractor(extractor_type: str, **kwargs):
```

Comment:
Can we make this a class method instead? It would help keep it cleaner and reduce the need for both a `TextExtractor` and `TextExtractorBase`

Reply:
Sure.

Reply:
Ended up just renaming to get_text_extractor. Since the factory method needs to import all of the implementations, you end up with some awkward circular-ish imports if you put it in the TextExtractor class.